azure-pipelines.yml

```diff
@@ -0,0 +1,60 @@
+name: das-forecasting-jobs
```

Comment:
This is a build number format, this should be removed and we should use Gitversion for build number generation